### DISCUSSION TITLE
Timelapse beep for each frame taken ?

### DISCUSSION THREAD

**Discussion by ulcotter** [gopro/labs#379] [Feature Requests]
Is there a way to have a Beep for each timelapse photo taken? like at the same time as the red flash?
I want to use it to do stop motion without having to manipulate phone or camera since I will manipulate messy stuff!

**Comment by dnewman-gpsw** [comment]
With HERO11 you could create a script for that.  However it would use the Photo mode, rather than timelapse mode.  There is a beep and blink command now, so you can trigger than, even moments before the shutter.

**Reply by ulcotter** [reply]
.. Thanks for your reply! I'm not (yet) familiar with scripting. I was hoping there's something somewhere I could use/tweak for this. Photo mode is fine. My plan b would be to use an app on my phone that emits a sound at the same frequency as the Timelapse intervals...

**Reply by dnewman-gpsw** [reply]
The camera defaults to beeping and blinking as each photo is taken so each the classic first generation scripting might work

![image](https://github.com/gopro/labs/assets/10856084/13ebd47b-db3c-4835-9e95-6639861849c8)

dPmP!9S!R  explained

dP -  default Photo (mode it will boot into)
mP - mode Photo (set current mode to Photo)
!9S - countdown 9 seconds before pressing Shutter (longer than 9 seconds than the camera will sleep, so dPmP!30S!R works too.)
!R - repeat.

With the newer scripting, you can get very fancy.

**Reply by ulcotter** [reply]
Thanks -0 I'll look into this!!
